src/error/format.rs

```diff
@@ -438,6 +438,9 @@ fn did_you_mean(styled: &mut StyledStr, context: &str, valid: &ContextValue) {
         styled.none(TAB);
         styled.good("note: ");
         styled.none(context);
+        if valid.len() > 1 {
+            styled.none("s");
+        }
```

Comment:
This feels really dirty to me to make assumptions on the passed in value like this

However, this is all internal and we can easily change it later and ideally new tests would be written for new cases added so it'd be noticed if its bad.